assets/css/index.css

```diff
@@ -1,75 +1,126 @@
-@import '/assets/css/reset.css';
-@import '/assets/css/nav.css';
-@import '/assets/css/footer.css';
-
+@import "/assets/css/reset.css";
+@import "/assets/css/nav.css";
+@import "/assets/css/footer.css";
 
 /* body & image section */
 
-.comp-lib-sub-body{
-    display: flex;
-    flex-direction: column;
-    justify-content: space-evenly;
-    flex-wrap: wrap;
-    padding-left: 3rem;
+.comp-lib-nav {
+  height: 100vh;
+  width: 100%;
+  background-color: var(--off-white);
+  transition: 0.4s ease-in-out;
 }
 
-.comp-lib-body{
-    display: flex;
-    align-items: center;
-    margin-left: 1rem;
-    margin-right: 1rem;
-    width: 100%;  
+.nav-icon {
+  display: flex;
+  justify-content: center;
+  align-items: center;
 }
 
-.comp-lib-body-ele{
-    display: flex;
-    flex-direction: column;
+#dark-mode {
+  /* margin-right: 2rem; */
+  cursor: pointer;
+}
+
+.comp-lib-nav-wrapper {
+  height: 100%;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+}
+
+.comp-lib-sub-body {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-evenly;
+  flex-wrap: wrap;
+  padding-right: 3rem;
+  padding-left: 3rem;
+}
+
+.comp-lib-body {
+  display: flex;
+  align-items: center;
+  margin-left: 1rem;
+  margin-right: 1rem;
+  width: 100%;
```

Comment:
Suggestion: We could use a short-hand property instead of mentioning them separately.

margin: 0 1rem;

It will help:
1. avoid writing another property,
2. On the next refactor,  it would be easy to modify in one place.